tests/unit/test_report_show_e2e.py

```
@@ -245,3 +245,92 @@ def capture_send(report_self):
+
+		captured = {}
+
+		def capture_send(report_self):
+			captured['results'] = report_self.data['results']
+
+		args = ['r', 'show', '-w', WORKSPACE, '--driver', 'local']
+		if report_query_arg:
+			args += [report_query_arg]
+
+		with mock.patch('secator.query.json.CONFIG') as mock_json_cfg, \
+				mock.patch('secator.query.utils.CONFIG') as mock_utils_cfg, \
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)